content/id/docs/concepts/storage/volume-snapshot-classes.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Volume Snapshot Class
+content_template: templates/concept
+weight: 30
+---
+
+{{% capture overview %}}
+
+Laman ini menjelaskan tentang konsep VolumeSnapshotClass pada Kubernetes. Sebelum melanjutkan,
+sangat disarankan untuk membaca [_snapshot_ volume](/docs/concepts/storage/volume-snapshots/)
+dan [kelas penyimpanan (_storage class_)](/docs/concepts/storage/storage-classes) terlebih dahulu.
+
+{{% /capture %}}
+
+
+{{% capture body %}}
+
+## Pengenalan
+
+Seperti halnya StorageClass yang menyediakan cara untuk para admin mendefinisikan
```

Comment:
...yang menyediakan cara bagi admin untuk mendefinisikan..

Reply:
fixed in 97c5357